views/partials/ideaMini.handlebars

```diff
@@ -0,0 +1,9 @@
+{{#each ideas}}
```

Comment:
With the update the how we fetch the data in the route, this should be updated to `{{#each space.ideas}}` or something similar. (Look at the actual `space` data structure by console logging it in the route.